src/datepicker/bs-datepicker.spec.ts

```diff
@@ -165,4 +165,24 @@ describe('datepicker:', () => {
         expect(view.date.getDay()).toEqual((new Date().getDay()), 'should update to equal today');
       }).unsubscribe();
   });
+
+
+  fit('should clear date', () => {
```

Comment:
please, remove **f**

Reply:
Sorry @daniloff200 
removed the **f**
Could you please help me to fix this test case for the clear date